### ISSUE TITLE
Ratio Clip

### ISSUE THREAD

**Issue by Leun9** [ChenxinAn-fdu/POLARIS#34]
Regarding the ratio clip issue, the blog mentions:
“Clip High (from DAPO): We increase the upper clipping bound in the surrogate loss function to encourage more aggressive exploration. This adjustment helps stabilize entropy and has been shown to improve model performance by allowing the policy to take larger, more beneficial update steps.”

However, based on the script and the wandb logs, it appears that an on‑policy update is being used (batch size == mini batch size), which means the clipping is effectively not taking effect.

**Comment by ChenxinAn-fdu** [comment]
In a purely on-policy setting, the ratio p θ / p_old is always 1.0, so the upper clipping does nothing. However, off-policy training and Polaris are orthogonal, so adding the upper clip here shouldn’t have any negative effect. We use the setting for researchers who use smaller `ppo_mini_batch_size`.
